pkg/odo/cli/url/create.go

```diff
@@ -97,8 +97,8 @@ func NewURLCreateOptions() *URLCreateOptions {
 }
 
 // Complete completes URLCreateOptions after they've been Created
-func (o *URLCreateOptions) Complete(_ string, cmd *cobra.Command, args []string) (err error) {
-	o.DevfilePath = clicomponent.DevfilePath
+func (o *URLCreateOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	o.PushOptions.CompleteDevfilePath()
```

Comment:
We can just do `o.CompleteDevfilePath()` like done here https://github.com/openshift/odo/pull/3857

Reply:
addressed